src/queries/user.ts

```diff
@@ -0,0 +1,21 @@
+import { createConnection, getConnection } from "typeorm";
+import { User } from "../entity/User";
+import * as bcrypt from "bcrypt";
+import * as jwt from "jsonwebtoken";
+import { ValidationError, BadCredentials, NotFound } from "../types-and-classes/errors";
+
+export const user = async (_, args, context) => {
+  jwt.verify(context.authScope, process.env.JWT_SECRET);
+  const user: any = (
+    await getConnection().manager.find("user", { id: args.id })
+  )[0];
```

Comment:
Usando repo, como costumamos fazer, é mais simples e mais legível:

```suggestion
  const user = await getRepository(User).findOne({ id: args.id });
```